src/ocamlorg_web/lib/templates/pages/package_template.eml

```diff
@@ -34,8 +34,8 @@ let render ~readme ~license:_ package =
     | "" -> gettext "No License"
     | l -> l
   in
-  let rev_deps = package_info.Ocamlorg.Package.Info.rev_deps |> List.map (fun (name, x) ->
-      (Ocamlorg.Package.Name.to_string name, x)
+  let rev_deps = package_info.Ocamlorg.Package.Info.rev_deps |> List.map (fun (name, version) ->
+      (Ocamlorg.Package.Name.to_string name, Ocamlorg.Package.t)
```

Comment:
I just assumed it would be something like this?

Reply:
You've turned `rev_deps` into a list of triples, so when mapping over it we will need to at least do something with each. 

```ocaml
let rev_deps = package_info.Ocamlorg.Package.Info.rev_deps |> List.map (fun (name, x, version) ->
      (Ocamlorg.Package.Name.to_string name, x, (* Convert version to a string here *))
```

Note you will also have to change the code further below that uses the `rev_deps`:

```ocaml
<% rev_deps |> List.iter begin fun (name, cstr, version) -> %>
```

Reply:
I see, thanks, and can I ask what does this 'x' mean? Using this when something is uncertain?

Reply:
`x` here is just a (pretty undescriptive 😅 )  variable name for the second element of the triple. The second element being the optional string describing the version constraints (e.g. `">= 4.3.0"`) -- later you see it given the slightly more descriptive `cstr` name.

Reply:
Great, thank you :)